contracts/bridge/BridgeTransferERC721.sol

```diff
@@ -110,7 +131,21 @@ contract BridgeTransferERC721 is BridgeTokens, IERC721BridgeReceiver, BridgeTran
     )
         public
     {
-        _requestERC721Transfer(msg.sender, _from, _to, _tokenId, _extraData);
+        uint V1 = 1;
```

Comment:
I think this is to be V2?

Reply:
This function is originally an existing function that I didn't touch except for an additional version parameter. The variable `V2` is located in `onERC721ReceivedV2()` function, which is called through `requestValueTransferV2()`.

Reply:
`onERC721Received` 
This is a standard interface of ERC-721. So if we leave this as it is, this function is anyway called and V1 will be executed even on the updated contract.

https://eips.ethereum.org/EIPS/eip-721

Reply:
That's my expected usage. An end-developer should call `RequestValueTransferV2()` which calls `onERC721Received2()` if he/she want to emit newly added event which fixes URI-loss problem. I think your expected usage seems the other way against my expected usage. Do you want me to change that `onERC721Received()` would call `_requestERC721Transfer()` with version 2?

Reply:
I think the `onERC721Received()` function should not call `requestValueTransferV2()` for a compatibility issue. Consider the following example.

- Mainbridge and subbridge nodes use the new version's binary and the old version's binary, respectively. (Or you are okay to think vice versa)
The new event will be emitted if the mainbridge node calls `requestValueTransfer()` on the ERC-721 contract. The subbridge node is not listening to the new event and nothing happens. Thus, I think we should keep the way of the current PR.